privacy/ccpa/policy.go

```diff
@@ -29,13 +36,20 @@ func ReadFromRequestWrapper(req *openrtb_ext.RequestWrapper, gpp gpplib.GppConta
 				}
 			}
 		}
+		if req.BidRequest.Regs.USPrivacy != "" {
+			if consent == "" {
+				consent = req.BidRequest.Regs.USPrivacy
```

Comment:
Can you add test coverage for this line?

Reply:
Test added